### PR TITLE
libs: Add version info script

### PR DESCRIPTION
**Pull request by mzidek-gh**
Add version info script for liabrt and libart_gui.

Fixes:
https://github.com/abrt/abrt/issues/603

Nore: We will still have soname 0.0.1 until next release (we will bump the version for every release).

### CODE REVIEW

File: src/configuration-gui/Makefile.am [hunk 1]
```diff
@@ -63,4 +68,4 @@ uidir = $(pkgdatadir)/ui
 EXTRA_DIST = abrt_gui.pc.in
 
 pkgconfigdir = $(libdir)/pkgconfig
-pkgconfig_DATA = abrt_gui.pc
```
Comment (mgrabovsky) [inline src/configuration-gui/Makefile.am:66]: Why was this removed?

**Review by mgrabovsky — COMMENTED** [review]
Looks good to me. Only need a clarification above.

**Comment by mgrabovsky** [comment]
Also, please add a changelog entry for the change.

File: src/configuration-gui/Makefile.am (hunk 1 above)
Comment (mzidek-gh) [reply src/configuration-gui/Makefile.am]: This was some kind of strange git rebase error. I checked my local original patch this change was not there, but after rebasing the patch on top of current master the line was removed. Not sure why git did that, but it was not intended. Thanks for the catch.

**Review by mgrabovsky — APPROVED** [review]
Looks good to me.
